README.md

```diff
@@ -0,0 +1,88 @@
+[![Test the action](https://github.com/canonical/craft-actions/actions/workflows/test.yml/badge.svg)](https://github.com/canonical/craft-actions/actions/workflows/test.yml)
+
+# Craft Actions
+
+This project contains custom GitHub actions for Craft Applications, such as:
+
+- [Charmcraft](https://juju.is/)
+- [Snapcraft](https://snapcraft.io/)
+- [Rockcraft](https://canonical-rockcraft.readthedocs-hosted.com/en/latest/)
+
+The following sections will break down the different actions available in this
+repository and how to use them.
+
+## rockcraft-pack
+
+Can be used to build a ROCK (with Rockcraft). It expects a "rockcraft.yaml" file
+to exist in its execution path.
+
+### Usage
+
+```yaml
+steps:
+  - uses: actions/checkout@v3
+  - uses: canonical/craft-actions/rockcraft-pack@main
+```
+
+This will install and configure LXD and Rockcraft, and then invoke `rockcraft`
+to build a new ROCK, based on a "rockcraft.yaml" file located in the root of
+the workingspace.
+
+You can also upload the resulting ROCK as a GitHub artifact as follows:
+
+```yaml
+steps:
+  - uses: actions/checkout@v3
+  - uses: canonical/craft-actions/rockcraft-pack@main
+    id: rockcraft
+  - uses: actions/upload-artifact@v3
+    with:
+      name: rock
+      path: ${{ steps.rockcraft.outputs.rock }}
+```
+
+Additionally, the following steps will already have Rockcraft available for
```

Comment:
```suggestion
Additionally, any following steps will already have Rockcraft available for
```

This one is minor: my original reading of "the following steps" was "the steps outlined next on this document" but I assume you mean "whatever additional steps that are added after these ones".